doc/getting_started/setup_dv.md

```diff
@@ -66,22 +66,23 @@ There is support for Cadence Xcelium as well, which is being slowly ramped up.
 The `uvmdvgen` tool provides an empty shell sequence at `dv/env/seq_lib/<dut>_sanity_vseq.sv` for developing the sanity test.
 The sanity test can be run as-is by invoking `dvsim.py`, as a "hello world" step to bring the DUT out of reset.
 ```console
-$ util/dvsim/dvsim.py path/to/<dut>_sim_cfg.hjson -i <dut>_sanity [--waves] [--tool xcelium]
+$ util/dvsim/dvsim.py path/to/<dut>_sim_cfg.hjson -i <dut>_sanity [--waves <format>] [--tool xcelium]
 ```
 
 The generated initial testbench is not expected to compile and elaborate successfully right away.
-There may be additional fixes required, which can be hopefully be identified easily.
+There may be additional fixes required, which can hopefully be identified easily.
 Once the testbench compiles and elaborates without any errors or warnings, the sanity sequence can be developed further to access a major datapath and test the basic functionality of the DUT.
 
 VCS is used as the default simulator.
 It can be switched to Xcelium by setting `--tool xcelium` on the command line.
-To dump waves from the simulation, pass the `--waves` argument to `dvsim.py`.
+To dump waves from the simulation, pass the `--waves <format>` argument to `dvsim.py`. With Xcelium you should probably use '--waves shm' and with VCS specify '--waves vpd' but both can generate the more portable, but much larger, VCD format by specifying the '--waves vcd' parameter.
```

Comment:
or `fsdb` for VCS if you using Verdi? @sriyerg

Reply:
Perhaps we could just list the supported formats and let the user pick whichever their setup supports (not everyone will have access to a Verdi license (sold separately by Snps). 

```suggestion
To dump waves from the simulation, pass the `--waves <format>` argument to `dvsim.py`. 
- With VCS, `vpd`, and `fsdb` are the supported formats.
- With Xcelium,  `shm`, `fsdb` and `vcd` are the supported formats.
- To find the list of supported formats for each tool, please lookup the `supported_wave_formats` key in the tool specific `dvsim.py` setup at `hw/dv/tools/dvsim/<tool>.hjson`.
```